openvpn/start.sh

```diff
@@ -92,9 +92,18 @@ function ufwAllowPortLong {
 }
 
 if [[ "${ENABLE_UFW,,}" == "true" ]]; then
+  if [[ "${UFW_DISABLE_IPTABLES_REJECT,,}" == "true" ]]; then
+    # A horrible hack to ufw to prevent it detecting the ability to limit and REJECT traffic
+    sed -i 's/return caps/return []/g' /usr/lib/python3/dist-packages/ufw/util.py
+    # force a rewrite on the enable below
+    echo "Disable and blank firewall"
+    ufw disable
+    echo "" > /etc/ufw/user.rules
+  fi
   # Enable firewall
   echo "enabling firewall"
   sed -i -e s/IPV6=yes/IPV6=no/ /etc/default/ufw
+  sed -i -e s/MANAGE_BUILTINS=no/MANAGE_BUILTINS=yes/ /etc/default/ufw
```

Comment:
Ahhh! Sorry, completely forgot about this change (I was thinking the first `sed`, which is what I spent ages figuring out initially). I did put that there, as if it's set, it will ensure that `ufw` will flush all built in tables as well - so if something else is messing with the firewall, it will be cleaned up (not likely as it's all in Docker, but possible if people are doing their own mixes and they install something that messes with the rules).

It's not required for this change to work, so if you don't think it should be there, I can just remove it altogether.

Cheers.

Reply:
Allright, we'll keep it for now and then we'll see later.